### DISCUSSION TITLE
why a safe exam browser need an internet connection to setup

### DISCUSSION THREAD

**Discussion by EztechTeam1** [SafeExamBrowser/seb-win-refactoring#840] [Q&A]
why a safe exam browser need an internet connection to setup
we work locally not at the internet and when I want to setup the app still shown this screen
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/146738178/79e855d3-44f2-469d-a98e-71dd7045480b)
I setup .net  & c++ & MicrosoftEdgeWebview2Setup 
why we need to the internet connection?
is there any way to setup safe exam browser without the internet

**Comment by dbuechel** [comment]
The setup bundle automatically downloads and installs the required runtime dependencies, it thus needs an internet connection. If you ensure that you install all required dependencies manually, then you can simply use the setup packages themselves (.msi) which only contain and install the SEB binaries.
